ghc-events.cabal

```diff
@@ -61,6 +61,8 @@ extra-source-files: include/EventLogFormat.h
                     test/hello-ghc-8.6.5.eventlog.reference
                     test/biographical-samples.eventlog
                     test/biographical-samples.eventlog.reference
+                    test/time-prof.eventlog
+                    test/time-prog.eventlog.reference
```

Comment:
s/prog/prof/. I think this is the culprit of the CI failures.

Reply:
Thanks.